tests/fixtures/parsers/typescript-parsers/array-callback-return.js

```diff
@@ -0,0 +1,320 @@
+"use strict";
+
+/**
+ * Parser: typescript-eslint-parser-3.0.0
```

Comment:
This is the parser that I set in https://astexplorer.net/
Is that what was expect of this comment format?